src/scripts/content/zendesk.js

```diff
@@ -27,6 +26,5 @@ togglbutton.render('.pane_header:not(.toggl)', { observe: true }, function (
     projectName: projectName && projectName.split(' - ').shift()
   });
 
-  divTag.appendChild(link);
-  elem.insertBefore(divTag, elem.querySelector('.btn-group'));
+  elem.insertBefore(link, elem.querySelector('.btn-group'));
```

Comment:
There's some problems with the TE popup and Zendesk

- Link always says "Start timer" even if timer is already running
- The title attribute is `undefined`

See link
https://streamja.com/2l4z